app/Http/Controllers/Auth/RegisteredUserController.php

```diff
@@ -40,12 +53,57 @@ public function store(Request $request): RedirectResponse
             'name' => $request->name,
             'email' => $request->email,
             'password' => Hash::make($request->password),
-        ]);
+        ])->assignRole(Role::findByName('user'));
 
-        event(new Registered($user));
+        // delete Invitation from new user
+        Invitation::where('email', $user->email)->first()->delete();
 
+        event(new Registered($user));
         Auth::login($user);
 
         return redirect(RouteServiceProvider::HOME);
```

Comment:
Nach Registrierung sollte m.E. eine Weiterleitung aufs Dashboard erfolgen, wenn sich ein Teacher registriert hat und eine Weiterleitung in die Admin-Area, wenn sich ein Admin registriert hat. Ich komme immer mehr zu der Auffassung, dass Teacher in der Admin-Area doch nichts verloren haben. In der Filament-Doku schaut es auch so aus, als wäre Filament nicht fest an eine Admin-Area gebunden. Lass uns dazu nochmal sprechen.